models/repo/repo.go

```diff
@@ -179,6 +180,7 @@ type Repository struct {
 	IsFsckEnabled                   bool               `xorm:"NOT NULL DEFAULT true"`
 	CloseIssuesViaCommitInAnyBranch bool               `xorm:"NOT NULL DEFAULT false"`
 	Topics                          []string           `xorm:"TEXT JSON"`
+	HashType                        string             `xorm:"NOT NULL DEFAULT 'sha1'"`
```

Comment:
The default size is 255, maybe it's too long.

Reply:
Thanks. I'll change this to CHAR(6)

Reply:
looks like this caused other issues .... so VARCHAR(6) it is!